src/github.rs

```diff
@@ -1505,6 +1530,40 @@ impl GithubClient {
             }
         }
     }
+
+    /// Returns whether or not the given GitHub login has made any commits to
+    /// the given repo.
+    pub async fn is_new_contributor(&self, repo: &Repository, author: &str) -> bool {
+        if repo.fork {
+            // GitHub always returns 0 results in forked repos, so this cannot
+            // work for them.
+            return false;
+        }
+        let url = format!(
+            "{}/search/commits?q=repo:{}+author:{}",
```

Comment:
The search API (is this the search API?) is heavily rate limited, so I'm not sure it's a good idea to call it here. Keeping a list of users in triagebot's db seems not too hard though, if we wanted to?

Reply:
Good point.  The rate limit on search is not too terrible (30/min for authenticated), which would be a pretty high rate of new PRs (and AFAIK hasn't bothered highfive in the past).

However, I just noticed that there is a `commits` endpoint which can filter on `author` which returns exactly the same results as the search API, but with the higher rate limit. I pushed a change which switches to that. Does that alleviate your concern? 

I'm a little worried that trying to track this in the db would cause a little disruption for existing repos (since it would either need to be prepopulated, or think everyone is new). Not a big deal, but it would also be a little more code.

Reply:
We're already using search API for lang team agendas, which are separately rate limited, so the usage here could cause problems with that use - since we are sharing the rate limit.

I agree that in general 30 req/min should be OK for this purpose since we're not seeing that high a PR rate, nor will we.

But seems like we have a better solution in any case :)